### PR TITLE
Adapt to new node-based respec2html.js

### PR DESCRIPTION
**Pull request by dontcallmedom**

### CODE REVIEW

File: generators/respec.js [hunk 1]
```diff
@@ -1,10 +1,8 @@
 
-var phantom = require("phantomjs")
-,   spawn = require("child_process").spawn
-,   jn = require("path").join
+var jn = require("path").join
```
Comment (deniak) [inline generators/respec.js]: `jn` is not used.

**Comment by deniak** [comment]
respec 3.2.119 is not yet available. I'll resume the review after it's published.

**Comment by dontcallmedom** [comment]
@deniak if you want to skip waiting for the release, you can use a clone of respec with PR 705 included (but I understand if you can't or don't want)

**Comment by deniak** [comment]
@dontcallmedom the PR works fine if the document is a respec document but in the other case I'm getting the following error
`Unhandled error in async function. [Error: .wait() timed out after 30000msec]`.

**Comment by dontcallmedom** [comment]
yeah, I already filed https://github.com/w3c/respec/issues/706 for this; but because of the relative urgency of getting this deployed, I figure we can leave error cases to a more relaxed release

**Comment by deniak** [comment]
Agree. Fill free to merge but we'll have to wait for the latest version of respec on npmjs and the node 5 of the servers.

**Pull request merged**